src/kits/tracker/FindPanel.cpp

```diff
@@ -650,6 +768,76 @@ FindWindow::MessageReceived(BMessage* message)
 			Save();
 			break;
 
+		case kOpenSaveQueryPanel:
+		{
+			BSaveWindow *saveQueryWindow = new BSaveWindow(this);
+			saveQueryWindow->Show();
+			break;
+		}
+
+		case kCloseSaveQueryPanel:
+		{
+			const char *queryName;
+			bool includeInFavorites;
+			bool saveInDefaultDirectory;
+
+			if(message->FindString("Query Name", &queryName) == B_OK
+				&& message->FindBool("Include In Favorites", &includeInFavorites) == B_OK
+				&& message->FindBool("Save In Default Directory", &saveInDefaultDirectory )== B_OK){
```

Comment:
space between ) and {

This is in multiple files.

Reply:
Fixed in the Latest commit.